net.audiorelay.AudioRelay.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: net.audiorelay.AudioRelay
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: AudioRelay
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --persist=.skiko
+  - --persist=.java
+
+modules:
+  - name: audiorelay
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 audiorelay.sh /app/bin/AudioRelay
+      - install -Dm755 apply_extra -t /app/bin
+      - install -Dm644 net.audiorelay.AudioRelay.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 net.audiorelay.AudioRelay.png -t /app/share/icons/hicolor/512x512/apps
+      - install -Dm644 net.audiorelay.AudioRelay.desktop -t /app/share/applications
+    sources:
+      - type: script
+        commands:
+          - exec /app/extra/bin/AudioRelay "$@"
+        dest-filename: audiorelay.sh
+
+      - type: script
+        commands:
+          - tar -xvf audiorelay.tar.gz
```

Comment:
silence output from tar here, it will leak into flatpak cli when installing.

Reply:
fixed